dot/parachain/types/types.go

```diff
@@ -556,3 +560,27 @@ type ValidatorSignature Signature
 
 // Signature represents a cryptographic signature.
 type Signature [64]byte
+
+// BackedCandidate is a backed (or backable, depending on context) candidate.
+type BackedCandidate struct {
```

Comment:
There is one more declaration of BackedCandidate https://github.com/ChainSafe/gossamer/blob/293c78cbe34030e1ce26f3cd98801c773779246d/lib/babe/inherents/parachain_inherents.go#L336

See if we can remove the other declaration and use this one everywhere

Reply:
I found multiple duplicate types.
I will raise separate PR to remove them all after this PR gets merged.